tests/test_models.py

```diff
@@ -16,6 +17,19 @@
     ConfigKey,
     ConfigScope,
 )
+from kirby.web import app_maker
+
+
+def test_database_config_exception():
```

Comment:
I tend to put the purpose of the test right into the test name, so this one would become `test_exception_is_raised_when_db_config_is_missing` for instance.